app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
@mcritchlow The reason to place at the 1st place: if it is .jpg file, it would override the mime type in metadata, because for all .jpg file we want it set as Content-Type: image/jpeg regardless how it is defined in metadata, doesn't it?